src/AppKit/XEnums.cs

```diff
@@ -7,26 +7,40 @@
 // Copyright 2012 Xamarin Inc
 
 using System;
+using System.Runtime.Versioning;
```

Comment:
Same, processed by the generator:

https://github.com/xamarin/xamarin-macios/blob/697e5b330e3a1fc9dd05146ccae28ef591a1ee79/src/frameworks.sources#L81